### PR TITLE
Add Css3test.com.xml

### PR DESCRIPTION
**Pull request by J0WI**

### CODE REVIEW

File: src/chrome/content/rules/Css3test.com.xml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<ruleset name="Css3test.com">
+	<target host="css3test.com" />
+	<target host="www.css3test.com" />
```
Comment (Bisaloo) [inline src/chrome/content/rules/Css3test.com.xml]: This target has an invalid certificate (`www.github.com`) but can be redirected to https://css3test.com/

**Comment by J0WI** [comment]
SSL Labs reports inconsistent results: https://www.ssllabs.com/ssltest/analyze.html?d=css3test.com

**Comment by Bisaloo** [comment]
As far as I can tell, it has been updated and now consistently fails for `www`. Could you have another look please?

The `unittests` test also consistently fails. You might need to rebase your branch on top of current master to fix it.

**Comment by J0WI** [comment]
Confirmed that the results are now consistent.

**Pull request merged**